Sources/StreamChatUI/ChatMessageList/Attachments/AttachmentViewCatalog.swift

```diff
@@ -35,6 +37,10 @@ open class AttachmentViewCatalog {
             return components.isVoiceRecordingEnabled
                 ? components.voiceRecordingAttachmentInjector
                 : components.filesAttachmentInjector
+        } else if attachmentCounts.keys.contains(.unknown) {
+            return components.unsupportedAttachmentInjector
+        } else if attachmentCounts.isEmpty == false {
+            return components.unsupportedAttachmentInjector
```

Comment:
why if there are no attachments we return the `unsupportedAttachmentInjector`?

Reply:
The condition is the other way around. If there are atttachments, but we don't know the type, then the Unsupported injector is used. (This won't impact customers customizations, since they have their own catalog logic, and if they find their custom type, they user their custom injector). The Demo App is testing all this with the Location attachment